project/BuildDependencies/scripts/0_package.list

```diff
@@ -7,8 +7,8 @@
 ; -> ...
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 bzip2-1.0.5-win32.7z
-crossguid-8f399e8bd4-win32.7z
-curl-7.42.1-win32.7z
+crossguid-8f399e-win32-vc140.7z
+curl-7.46.0-win32-vc140.7z
```

Comment:
I can rebuild with latest curl, openssl is used by something else as well so that will be kept on current version. 

We have used statically linked curl before and it works fine in my testing